### ISSUE TITLE
[Bug]: Binding WCF service endpoint to empty address blocks access to other endpoints and returns 400

### ISSUE THREAD

**Issue by LucDD** [CoreWCF/CoreWCF#1315]
### Duplicate ?

- [X] I have searched issues/discussions and did not find other issues/discussions reporting this bug.

### Product version

1.5.1

### Describe expected behavior

Using the CoreWCF Visual Studio project template.
Bind WCF service to address "/"
MapGet "Hello world!" to address "/hello"
Accessing "https://localhost:{devport}/hello" should display "Hello world!" text in the browser.

This is the behaviour in CoreWCF versions 1.2.1 and earlier.

### Describe actual behavior

Browser displays HTTP ERROR 400

Occurs in all versions of CoreWCF >= 1.3.0

### Which binding

BasicHttp

### security

Transport

### Which .NET version

.NET 6

### Which os platform

Windows

### Code snippet used to reproduce the issue

```c#
var builder = WebApplication.CreateBuilder(args);

builder.Services.AddServiceModelServices();

var app = builder.Build();

app.MapGet("/hello", () => "Hello World!");

app.UseServiceModel(serviceBuilder =>
{
    serviceBuilder.AddService<Service>();
    serviceBuilder.AddServiceEndpoint<Service, IService>(new BasicHttpBinding(BasicHttpSecurityMode.Transport), "/");
});

app.Run();
```

**Comment by mconnew** [comment]
Are you sure the MapGet before the call to UseServiceModel as CoreWCF participates like any other middleware so if MapGet comes first, it will get the first look at the request and handle it if necessary. I would expect the behavior you are seeing if MapGet comes after UseServiceModel but not before.

**Comment by LucDD** [comment]
Yeah, that's what I was expecting too but it doesn't seem to matter that the MapGet call is first though. The code snippet above is what I had in Program.cs. The rest (Service/IService) is the standard stuff from the CoreWCF Visual Studio project template.

**Comment by ctechid** [comment]
New ASP.NET Core projects are configured to bind to a random HTTP port between 5000-5300 and a random HTTPS port between 7000-7300. 

The selected ports are stored in the generated **Properties/launchSettings.json** file and can be modified by the developer. The **launchSetting.json file** is only used in local development.

If there's no endpoint configuration, then Kestrel binds to **http://localhost:5000.**

Look at https://learn.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel/endpoints?view=aspnetcore-8.0

Try to access
https://localhost:7000/hello
https://127.0.0.1:7000/hello

http://localhost:5000/hello
http://127.0.0.1:5000/hello

Or look your port on **Properties/launchSettings.json** maybe it has other values.

**Comment by LucDD** [comment]
I don't think the port matters, other than that the desired behaviour is to have the CoreWCF endpoint and MapGet (or regular controller) endpoint coexist on the same port (with CoreWCF being on the root, and other endpoints being on other paths).

In the example code I posted above, the /hello endpoint immediately works when the address in AddServiceEndpoint is changed from "/" to "/anything" (it also works when downgrading the version of CoreWCF to 1.2.1 or lower).
